e2e/tests/functional/plugins/notebook/tags.e2e.spec.js

```diff
@@ -110,6 +110,24 @@ test.describe('Tagging in Notebooks @addInit', () => {
         await expect(page.locator('[aria-label="Autocomplete Options"]')).not.toContainText("Driving");
         await expect(page.locator('[aria-label="Autocomplete Options"]')).toContainText("Drilling");
     });
+    test('Can add tags with blank entry', async ({ page }) => {
+        //Go to baseURL
+        await page.goto('./', { waitUntil: 'networkidle' });
```

Comment:
usually this is common to all beforeEach steps

Reply:
It's true, but the tests above are also adding notebook entries with text, which we don't want for this particular test.